<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AblationPreset</class>
 <widget class="QWidget" name="AblationPreset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1372</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AblationPreset</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(12, 75, 153);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>1241</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1,14,2,0">
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="CatheterLayout" stretch="2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>4</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Catheter</string>
           </property>
           <property name="foreground">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>ThermoCool®</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>ThermoCool® SF                                  </string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>4mm</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>8mm</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,1,9">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddCatheter">
            <property name="styleSheet">
             <string notr="true">font: 75 14pt &quot;Arial Narrow&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/add.jpg</normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/add.jpg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDeleteCatheter">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/delete.jpg</normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/delete.jpg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="PresetLayout" stretch="2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>4</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Preset                                                    (Default)</string>
           </property>
           <property name="foreground">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </column>
          <column>
           <property name="text">
            <string>MODE</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>TEMP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PWR</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>ThermoCool® Standard                             (D)</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>T</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>43℃</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string>20w</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>PHYSICIAN PWR CTRL</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>P</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string>N/A</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="1" column="3">
           <property name="text">
            <string>20w</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>PHYSICIAN 50W 40DEG</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string>T</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="2" column="2">
           <property name="text">
            <string>40℃</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="2" column="3">
           <property name="text">
            <string>50w</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>PHYSICIAN 70W 45DEG</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string>T</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="3" column="2">
           <property name="text">
            <string>45℃</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="3" column="3">
           <property name="text">
            <string>70w</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1,1,9">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddList">
            <property name="styleSheet">
             <string notr="true">font: 75 14pt &quot;Arial Narrow&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/add.jpg</normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/add.jpg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDeleteList">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/delete.jpg</normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/delete.jpg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="PresetButtonsLayout" stretch="0,0,0,0,0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="btnPresetEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/资源 7.png</normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/资源 7.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="btnPresetSetting">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/资源 5.png</normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/资源 5.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QPushButton" name="btnPresetConfirm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/资源 26.png</normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/资源 26.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="btnUSB1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/资源 11.png</normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/资源 11.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QPushButton" name="btnUSB2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/资源 12.png</normaloff>D:/UGit/RPFASystem/src/viewer/res/images/PNG/资源 12.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
